src/excmds.ts

```diff
@@ -751,6 +751,26 @@ export async function source_quiet(...fileArr: string[]) {
     }
 }
 
+/** Use tridactylrc located at the given url
+ * @param url the url where the raw version of tridactylrc can be found
+ */
+//#background
+export async function source_from_url(url: string) {
```

Comment:
I think I'd be happier if this was combined with the current `source` command, with this branch executed if there was a `--url` flag, e.g. `source --url news.bbc.co.uk`

Reply:
Sure. Will fix this.

Reply:
Probably worth having `source --url` with no args or `source --url %` source the current page, too :)

Reply:
Tridactyl doesn't work on "raw" files(at least for me). So, a `source --url` without an url is not required.

Reply:
It works if you `set csp clobber`, which you shouldn't do unless you understand what it does. It's a Firefox bug anyway which Mozilla is fixing, albeit very slowly, so I would be happier if we supported it. It should fetch the URL rather than page content as otherwise you'd have to work around the mode indicator etc.

Reply:
Thank you. Adding this now.